Client/TeamTalkClassic/TeamTalk.rc

```diff
@@ -270,6 +270,12 @@ BEGIN
     CONTROL         "",IDC_SLIDER_MEDIASTREAM_VOL,"msctls_trackbar32",TBS_BOTH | TBS_NOTICKS | WS_TABSTOP | 0x400,117,176,131,15
     PUSHBUTTON      "Refresh devices",IDC_BUTTON_REFRESHSND,132,193,60,12
     PUSHBUTTON      "&Default",IDC_BUTTON_DEFAULT,197,193,51,12
+    GROUPBOX        "Client sounds events settings",IDC_STATIC_GRPCLIENTSOUNDSYSTEM,200,7,246,207
+    CONTROL         "Play all sounds simultaneously",IDC_RADIO_PBMODETT,"Button",BS_AUTORADIOBUTTON | WS_TABSTOP,41,22,95,10

```

Comment:
There's no room for all these radio buttons. We should use a combobox with a droplist instead. You can find an example of this in Display-tab where you choose how to sort channels.

I can also look into it but it will take a few days before I can look at it.